plugin/helpers_kv.go

```diff
@@ -5,30 +5,19 @@ package plugin
 
 import (
 	"encoding/json"
-
 	"github.com/pkg/errors"
 )
 
-// KVGetJSON is a wrapper around KVGet to simplify reading a JSON object from the key value store.
-func (p *HelpersImpl) KVGetJSON(key string, value interface{}) (bool, error) {
```

Comment:
Why did you changed the ordering?

Reply:
The ordering was changed to match the declaration and implementation order to improve readability. I was given same feedback in one of the PR.

Reply:
You can check here: https://github.com/mattermost/mattermost-server/pull/12539#discussion_r333158417

Reply:
Good point and thanks for fixing this :+1: